### DISCUSSION TITLE
token expiration time

### DISCUSSION THREAD

**Discussion by 1392273211** [cobbler/cobbler#3636] [Q&A] [answered]
I see that Cobbler's token is currently 30 minutes long. Can I delay the token time? Where to modify

**Comment by SchoolGuy** [comment, marked as answer]
Yes you can. Please take a look at the documentation.

Source:
- `main` Branch: https://cobbler.readthedocs.io/en/latest/cobbler-conf/settings-yaml.html#auth-token-expiration
- 3.3.4: https://cobbler.readthedocs.io/en/v3.3.4/cobbler-conf.html#auth-token-expiration
- 3.3.3: https://cobbler.readthedocs.io/en/v3.3.3/cobbler-conf.html#auth-token-expiration
- 3.2.2: https://cobbler.readthedocs.io/en/v3.2.2/cobbler-conf.html#auth-token-expiration

**Reply by 1392273211** [reply]
Thank you.

**Reply by 1392273211** [reply]
I got an error when I changed the token to 0. Can't I set it to never expire?

**Reply by SchoolGuy** [reply]
No this is not possible as the tokens are stored in memory and as such get invalidated at minimum during a daemon restart. Furthermore it is not advisable to have temporary tokens with an indefinite amount of time.

**Reply by 1392273211** [reply]
Can it be modified to a limited period of 1 year?

**Reply by SchoolGuy** [reply]
Well of course but it would require that the daemon is not restarted for the whole year. The token is just kept in memory.

**Reply by 1392273211** [reply]
Thanks
